src/main/resources/templates/fragments/icons.html

```diff
@@ -780,7 +945,7 @@ <h1>Icons</h1>
                   stroke-linecap="round" stroke-linejoin="round"/>
           </g>
           <defs>
-            <clipPath id="clip0">
```

Comment:
Why did `clip0` need to be updated? Is there a `clip0` in a different icon?

Reply:
Yeah, clip0 appears a few times in here. We ran into an issue trying to display it on the same page